lib/hexo/index.js

```diff
@@ -74,17 +74,17 @@ const createLoadThemeRoute = function(generatorResult, locals, ctx) {
   };
 };
 
-function debounce(func, wait, immediate) {
+function debounce(func, wait) {
   let timeout;
   return function() {
     const context = this;
     const args = arguments;
     clearTimeout(timeout);
     timeout = setTimeout(() => {
       timeout = null;
-      if (!immediate) func.apply(context, args);
+      func.apply(context, args);
     }, wait);
-    if (immediate && !timeout) func.apply(context, args);
+    if (!timeout) func.apply(context, args);
```

Comment:
Since there's no `immediate` is given in actual practice, should we just remove this line as it is always false?

Reply:
indeed, `timeout` is never falsy.